### ISSUE TITLE
Accuracies for Model training and testing

### ISSUE THREAD

**Issue by sreedharjinka** [sreedharjinka/Thyroid-DetectionG97#14]

**Comment by Dhruv69408012** [comment]
The model training and testing are done using the sklearn package in python this package consists of various tools which help quantify the efficiency of the model and aid in testing the same. The accuracy_score function and train_test_split function help with the quantification the model itself was made using the xgboost module as mentioned in the problem statement and the accuracy yield is around 98% which is considered a good accuracy score according to machine learning standards.
[code.txt](https://github.com/sreedharjinka/Thyroid-DetectionG97/files/13264697/code.txt)
